OrbitClientServices/ProcessManager.cpp

```diff
@@ -23,6 +24,9 @@ class ProcessManagerImpl final : public ProcessManager {
  public:
   explicit ProcessManagerImpl(const std::shared_ptr<grpc::Channel>& channel,
                               absl::Duration refresh_timeout);
+  ProcessManagerImpl(ProcessManagerImpl&&) = default;
+  ProcessManagerImpl& operator=(ProcessManagerImpl&&) = default;
+  ~ProcessManagerImpl() override { Shutdown(); }
```

Comment:
nit: wanted to do that but while you are here maybe rename Shutdown() to ShutdownAndWait()?

Reply:
Also are we sure it does not throw bad_alloc? Maybe make it nothrow as well.

Reply:
I am not completely sure if I understood the nothrow part. I made the function noexcept now. Please take a look